tests/integration/test_cloudformation.py

```diff
@@ -1060,6 +1163,71 @@ def test_deploy_stack_with_sns_topic(self):
         topics = [tp for tp in rs['Topics'] if tp['TopicArn'] == topic_arn]
         self.assertEqual(len(topics), 0)
 
+    def test_deploy_stack_with_dynamodb_resource(self):
+        stack_name = 'stack-%s' % short_uid()
+        change_set_name = 'change-set-%s' % short_uid()
+        env = 'Staging'
+        ddb_table_name_prefix = 'ddb-table-%s' % short_uid()
+        ddb_table_name = '{}-{}'.format(ddb_table_name_prefix, env)
+
+        cloudformation = aws_stack.connect_to_service('cloudformation')
+
+        rs = cloudformation.create_change_set(
+            StackName=stack_name,
+            ChangeSetName=change_set_name,
+            TemplateBody=TEST_DEPLOY_BODY_3,
+            Parameters=[
+                {
+                    'ParameterKey': 'tableName',
+                    'ParameterValue': ddb_table_name_prefix
+                },
+                {
+                    'ParameterKey': 'env',
+                    'ParameterValue': env
+                }
+            ]
+        )
+        self.assertEqual(rs['ResponseMetadata']['HTTPStatusCode'], 200)
+        change_set_id = rs['Id']
+
+        rs = cloudformation.execute_change_set(
+            StackName=stack_name,
+            ChangeSetName=change_set_name
+        )
+        self.assertEqual(rs['ResponseMetadata']['HTTPStatusCode'], 200)
+
+        rs = cloudformation.describe_stacks(
+            StackName=stack_name
+        )
+        self.assertEqual(rs['ResponseMetadata']['HTTPStatusCode'], 200)
+
+        stacks = [stack for stack in rs['Stacks'] if stack['StackName'] == stack_name]
+        self.assertEqual(len(stacks), 1)
+        self.assertEqual(stacks[0]['ChangeSetId'], change_set_id)
+
+        outputs = {
+            output['OutputKey']: output['OutputValue']
+            for output in stacks[0]['Outputs']
+        }
+        self.assertIn('Arn', outputs)
+        self.assertEqual(outputs['Arn'], 'arn:aws:dynamodb:{}:{}:table/{}'.format(
+            DEFAULT_REGION, TEST_AWS_ACCOUNT_ID, ddb_table_name))
```

Comment:
Instead of importing `DEFAULT_REGION` directly, better use `aws_stack.get_region()` here.